app/src/main/java/com/jerboa/api/Http.kt

```diff
@@ -316,9 +316,12 @@ interface API {
 }
 
 sealed class ApiState<out T> {
-    class Success<T>(val data: T) : ApiState<T>()
+
```

Comment:
I still don't fully understand also why you can't add a `Refreshing` state here, instead of having isRefreshing in all the viewmodels.